### PR TITLE
4016 twiss report display

### PR DESCRIPTION
**Pull request by gurhar1133**
twiss report now in beamline panel and animates during optimization

### CODE REVIEW

File: sirepo/package_data/static/html/madx-visualization.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
     </div>
     <div class="clearfix"></div>
     <div data-ng-repeat="item in visualization.outputFiles" data-column-for-aspect-ratio="{{ item.modelAccess.modelKey }}" data-ng-if="visualization.simState.hasFrames()">
+      {{item.modelAccess}}
```
Comment (moellep) [inline sirepo/package_data/static/html/madx-visualization.html:13]: I think this was debugging in the html and can be removed.

File: sirepo/package_data/static/js/controls.js [hunk 2]
```diff
@@ -314,37 +320,80 @@ SIREPO.app.controller('ControlsController', function(appState, controlsService,
...
+            return;
+        }
+        const keys = [];
+        const modelKey = 'instrumentAnimationTwiss';
+        const modelViewName = 'instrumentAnimationTwiss';
+        const id = appState.models.externalLattice.models.length + 2;
+        keys.push(modelKey);
+        setAnimationModel(modelKey, modelViewName, id);
+        appState.saveChanges(keys);
+    }
```
Comment (moellep) [inline sirepo/package_data/static/js/controls.js:380]: I think all the "const xxx" values should be removed and placed within the calls. ex. setAnimationModel('instrumentAnimationTiwss', 'instrumentAnimationTwiss', appState.models.externalLattice.models.length + 2);

File: sirepo/template/controls.py [hunk 3]
```diff
@@ -67,8 +76,10 @@ def sim_frame(frame_args):
 
 def _extract_report_elementAnimation(frame_args, run_dir, filename):
     data = frame_args.sim_in
-    if sirepo.template.madx.is_parameter_report_file(filename):
-        return sirepo.template.madx.extract_parameter_report(data, run_dir, filename)
+    if 'twiss' in frame_args.frameReport.lower():
```
Comment (moellep) [inline sirepo/template/controls.py:79]: I think there is only one twiss report so you can be explicit here:
if frame_args.frameReport == 'instrumentAnimationTwiss':
    ...

**Comment by gurhar1133** [comment]
Made the changes

**Review by moellep — APPROVED** [review]

**Pull request merged**